app/models/form.rb

```diff
@@ -52,7 +52,7 @@ def name=(val)
   end
 
   def as_json(options = {})
-    options[:methods] ||= %i[live_at start_page has_draft_version has_live_version has_routing_errors]
+    options[:methods] ||= %i[live_at start_page has_draft_version has_live_version has_routing_errors ready_for_live incomplete_tasks task_statuses]
```

Comment:
I'm not sure I like this, it feels like it might pollute the form object computed data too much...

Reply:
does it make sense because the API is mainly about **editing** forms rather than live forms?

I guess live forms don't need this info and the options are overridden to not include it in those?

Reply:
When you put it like that, yes I suppose the sole purpose of this is to serve forms-admin what it needs. I just don't want to see an ever expanding list of dynamic attributes that get passed back as JSON. 😅 

Yeah the added dynamic attributes aren't included in the JSON snapshot made when forms are made lives 👍